### PR TITLE
Brush up some bestiary automation for Howl of the Wild

### PR DESCRIPTION
**Pull request by simonward**
T-Z

### CODE REVIEW

File: packs/howl-of-the-wild-bestiary/tyrafdir.json [hunk 1]
```diff
@@ -178,14 +198,38 @@
...
+                            }
+                        },
+                        "key": "Strike",
+                        "label": "PF2E.NPCAbility.Tyrafdir.DeceptiveJaws",
+                        "predicate": [
+                            "self:effect:deceptive-heads"
+                        ],
+                        "slug": "deceptive-jaws",
+                        "traits": [
+                            "mental",
```
Comment (simonward) [inline packs/howl-of-the-wild-bestiary/tyrafdir.json:227]: System doesn't currently support `illusion` trait for NPC attacks.

File: static/lang/re-en.json [hunk 2]
```diff
@@ -1101,6 +1104,11 @@
                     "DamageType": "Animated Weapon - Damage Type"
                 }
             },
+            "Werecreature": {
+                "Human": "Human",
+                "Moose": "Moose",
+                "Shark": "Shark"
+            },
```
Comment (SpartanCPA) [inline static/lang/re-en.json:1111]: These should be under PF2E.NPCAbility.ChangeShape.Form

Comment (simonward) [reply static/lang/re-en.json]: Good call.

File: packs/howl-of-the-wild-bestiary/wereshark.json [hunk 3]
```diff
@@ -417,7 +417,87 @@
...
                 },
-                "rules": [],
+                "rules": [
+                    {
+                        "domain": "all",
+                        "key": "RollOption",
+                        "option": "change-shape",
+                        "suboptions": [
+                            {
+                                "label": "PF2E.NPCAbility.ChangeShape.Form.Human",
```
Comment (stwlam) [inline packs/howl-of-the-wild-bestiary/wereshark.json:427]: Reuse existing localization keys

**Pull request merged**